MonstieWash/Assets/Scripts/Traversal/FinishLevelButton.cs

```diff
@@ -4,6 +4,9 @@ public class FinishLevelButton : MonoBehaviour, INavigator
 {
     public void OnClicked()
     {
-        GameSceneManager.Instance.FinishLevel();
+        DecorationManager dm = FindFirstObjectByType<DecorationManager>();
```

Comment:
### 🗒️ **Convention** | _This is for maintaining our coding conventions._

This can be using `var`.

```suggestion
        var dm = FindFirstObjectByType<DecorationManager>();
```